docs/src/modules/components/prism.js

```diff
@@ -14,8 +14,8 @@ let lightTheme;
 let darkTheme;
 
 if (process.browser) {
-  lightTheme = require('prismjs/themes/prism.css');
-  darkTheme = require('prismjs/themes/prism-okaidia.css');
+  lightTheme = require('prism-themes/themes/prism-ghcolors.css');
```

Comment:
Something is off, the readability has decreased:

Before

![Capture d’écran 2019-05-20 à 18 59 25](https://user-images.githubusercontent.com/3165635/58038878-764ecb00-7b31-11e9-9e74-2c2f3825c721.png)


After
![Capture d’écran 2019-05-20 à 18 59 17](https://user-images.githubusercontent.com/3165635/58038877-75b63480-7b31-11e9-8e82-59b7b390162e.png)

Reply:
I have noticed a weird underline on the dark theme:

**Before**
![Capture d’écran 2019-05-20 à 19 04 00](https://user-images.githubusercontent.com/3165635/58039126-1d336700-7b32-11e9-9b01-a11136f6c654.png)



**After**
![Capture d’écran 2019-05-20 à 19 03 56](https://user-images.githubusercontent.com/3165635/58039124-1d336700-7b32-11e9-875c-76dcedbe7231.png)